_saas-integrations/shopify/v1/shopify-v1.md

```diff
@@ -66,6 +66,9 @@ feature-summary: |
 #      Setup Instructions    #
 # -------------------------- #
 
+requirements-info: |
+  Staff members on a {{ integration.display_name }} Plus plan cannot export data by default. Store owners can give staff members permissions export orders, draft orders, products, inventory, and customer data. Read the [{{ integration.display_name }} Staff permissions documentation](https://help.shopify.com/en/manual/your-account/staff-accounts/staff-permissions#store-owner-permissions){:target="new"} for more information.
```

Comment:
Consider re-arranging some of this so that it matches the structure of requirements in other integrations. By doing this, you can also make it clear in the first sentence that the requirement is applicable only to a subset of Shopify users:

```
**If you're on a {{ integration.display_name }} Plus plan**, [...remaining content...]
```